### ISSUE TITLE
The app has a red light in Plank when launched

### ISSUE THREAD

**Issue by Loreno10** [elementary/mail#524]
I often see this:
![Screen record from 2020-12-16 21 29 09](https://user-images.githubusercontent.com/10273406/102403020-f37c4380-3fe5-11eb-9b76-a21176f737bf.gif)

I try to open Mail and it flashes red. I have to click it again, to actually open the app.
It does not happen all the time.

**Comment by Loreno10** [comment]
Additionally, the cursor turns to the "loading" state, which is also a bit confusing.

**Comment by hanaral** [comment]
I'm certain this is a plank issue, but I'm just as certain that it's how plank shows that something is still being opened/loaded

**Comment by Loreno10** [comment]
Well, feel free to move this issue to Plank repo. I experience this everyday, only with Mail.

**Comment by hanaral** [comment]
> Well, feel free to move this issue to Plank repo. I experience this everyday, only with Mail.

I meant in the sense that it _is_ in fact a feature, that was only implemented by a couple for apps. Either way, the problem will no longer exist in elementary 6 since they are replaced with monochrome dots.

**Comment by Loreno10** [comment]
Hmm, I have no idea about that. It sounds like it just changes the look of the dot to be monochrome? What about the fact that I have to click the Mail's icon twice to actually open its window?

**Comment by hanaral** [comment]
> Hmm, I have no idea about that. It sounds like it just changes the look of the dot to be monochrome? 

No, it changes the status indicator, and removes the light effect apps get if they open while the window is obfuscated.

>What about the fact that I have to click the Mail's icon twice to actually open its window?

I'd say that's a bug, but I'm not able to reproduce it. I assume the bug is resolved in elementary OS 6.

**Comment by Loreno10** [comment]
> I'd say that's a bug, but I'm not able to reproduce it. I assume the bug is resolved in elementary OS 6.

Ok, so hopefully I'll be able to verify that "soon", when eOS 6 is released.

**Comment by casasfernando** [comment]
I noticed @Loreno10 that seem to be using a custom Plank theme, rather than the default.

I faced the same issue after changing Plank's theme from "Gtk+" to a custom one downloaded from the Internet "[mcOS-BS-Black-Stock](https://www.gnome-look.org/p/1399398/)".
If I roll back to Plank's "Gtk+" or "Default" theme the problem is gone.

More over I just had the same issue while installing something in Wine (that opened a new Wine window) while writing this comment so I would say that this may be more of a theme issue rather than Plank or Mail.
Don't now how to troubleshoot it tough.

Just my two cents.
